discord/guild.py

```diff
@@ -949,7 +949,7 @@ def owner(self) -> Member | None:
 
     @property
     def icon(self) -> Asset | None:
-        """Returns the guild's icon asset, if available."""
+        """Returns the guild's icon asset of type :class:`asset.Asset`, if available."""
```

Comment:
Can this not use the `rtype` directive? 
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format

Reply:
Something like :class:`~discord.Asset` should work fine. But I can't see any place in the PyCord docs where poperty return  types are mentioned in the docstring.

Reply:
@mccoderpy https://discord.com/channels/858089281214087179/1055198103369682966/1121542979769880637

Reply:
Wouldn't it be :type:

Reply:
No, :type: isn't a thing by default, and types are mostly classes ^^

Reply:
`:type:` directive is to specify the type of a parameter. `:rtype` is to specify the return type.

See this https://www.sphinx-doc.org/en/master/tutorial/describing-code.html#documenting-python-objects